content/Certification/certifying_a_product/development.md

```diff
@@ -0,0 +1,197 @@
+
+-   TODO: need a way to address custom flow and the different requirements for
+    on-boarding material
+
+In addition to DACs, each individual unit needs to be provisioned with its own
+on-boarding material for discovery and initial commissioning. This includes the
+following per-unit items provisioned on the device:
+
+-   discriminator
+-   PAKE verifier (encodes the passcode and PAKE salt)
+-   PAKE salt
+
+It also includes the following items encoded on the device exterior or
+packaging, both of which encode the discriminator and passcode for the
+corresponding unit:
+
+-   QR code
+-   manual code
+
+Manual codes are required for all devices, QR codes are optional but
+recommended. The rules for QR and manual code inclusions are covered in section
+5.7.6 and section 13.6 of the specification. These requirements are attested at
+certification in the
+
+[Matter Setup Code Guidelines](https://groups.csa-iot.org/wg/members-all/document/27481)
+
+### Multi-product QR codes
+
+For products where multiple devices are in one box, it can be challenging for
+users to properly match individual on-boarding materials to the physical
+products in the box. In such situations, manufacturers should follow the
+guidance in the [Multi Device Setup Codes Best Practices Guide]().
+
+### In-field updates to Matter
+
+If the software being developed is for a product already in the field, the
+Matter firmware will need to distributed as an over-the-air update using the
+current update mechanism on the device. In-field update considerations are
+discussed in section 5.8 of the specification, but it is important to note that
+in-field products are subject to the same per-unit requirements as Out-of-Box
+Natter products, and thus it is important to consider how to distribute not only
+the firmware, but also how to display the on-boarding material or how to do a
+background update using the current connected application and how to provision
+the per-unit materials discussed earlier in this section in a secure manner
+using the security materials available to the current firmware.
+
+## Certification programs
+
+Product developers should be aware of the various certification programs and how
+they may impact the product development process across the entire line.
+
+In particular:
+
+-   Products considering software component certification should understand
+    which types of products and operating environments qualify for this program.
+-   Products with various SKUs and variants should understand the portfolio
+    certification program and what constitutes a viable portfolio parent.
+-   Bridge devices should understand the bridge certification program and how
+    certification of the supported device types works.
+
+Product manufacturers should be aware that the CSA has a sunset policy in effect
+for prior specification revisions. The sunset policy is available in full at
+(TODO: where's the sunset policy?).
```

Comment:
@amolivo - this is an SC thing right? Is this actually published anywhere?

Reply:
The dates are recommended by the SC and approved by the BoD. As of right now, there is no one place where the sunset dates/policy is held. I've asked for them to be included in the policy doc, itself, as a single point of reference. I'll re-raise this request in TCOC, as I think it has fallen off the proverbial radar.